### PR TITLE
feat: minor improvements to login UI

### PR DESCRIPTION
**Pull request by scmmishra**
This PR adds the following changes

1. Update login layout to center the login box
2. Add macOS like wiggle animation to login box on error
3. Update create account copy

### Demo

https://github.com/chatwoot/chatwoot/assets/18097732/8fdecbeb-35ec-4c49-a26d-6a0eacb29b2e

### CODE REVIEW

File: app/javascript/v3/views/login/Index.vue [hunk 1]
```diff
@@ -193,9 +200,46 @@ export default {
...
+          this.loginApi.hasErrored = true;
           this.showAlert(response?.message || this.$t('LOGIN.API.UNAUTH'));
         });
     },
   },
 };
 </script>
+
+<style scoped>
+.login--error {
```
Comment (scmmishra) [inline app/javascript/v3/views/login/Index.vue:212]: @pranavrajs let me know what you think of this animation

File: app/javascript/v3/views/login/Index.vue [hunk 2]
```diff
@@ -27,13 +27,16 @@
       >
         {{ $t('COMMON.OR') }}
         <router-link to="auth/signup" class="text-link">
-          {{ $t('LOGIN.CREATE_NEW_ACCOUNT') }}
+          {{ $t('LOGIN.CREATE_NEW_ACCOUNT').toLocaleLowerCase() }}
```
Comment (pranavrajs) [inline app/javascript/v3/views/login/Index.vue:30]: Let us CSS lowercase.

File: app/javascript/v3/views/login/Index.vue (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/v3/views/login/Index.vue:212]: Animation looks fine. Let us not write classes like this. Can we rewrite this in tailwind helpers?

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Looks good. Some minor comments.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**